modules/ROOT/pages/clauses/listing-functions.adoc

```diff
@@ -69,6 +69,11 @@ a| Whether the function is deprecated.
 label:new[Introduced in 5.9]
 m| BOOLEAN
 
+m| deprecatedBy
+a| The replacement function to use in case of deprecation.
```

Comment:
would probably be nice to mention t being null otherwise here as well (and maybe something about it potentially being null if no replacement is given even if deprecated)